pkgs/by-name/co/composefs/package.nix

```diff
@@ -45,27 +50,45 @@ stdenv.mkDerivation (finalAttrs: {
     (lib.enableFeature enableValgrindCheck "valgrind-test")
   ];
 
-  nativeBuildInputs = [ autoreconfHook go-md2man pkg-config ];
-  buildInputs = [ openssl ]
+  nativeBuildInputs = [
+    autoreconfHook
+    go-md2man
+    pkg-config
+  ];
+  buildInputs =
+    [ openssl ]
     ++ lib.optional fuseSupport fuse3
-    ++ lib.filter (lib.meta.availableOn stdenv.hostPlatform) (
-    [
+    ++ lib.filter (lib.meta.availableOn stdenv.hostPlatform) ([
       libcap
       libseccomp
-    ]
-  );
+    ]);
 
   doCheck = true;
-  nativeCheckInputs = [ python3 which ]
+  nativeCheckInputs =
+    [
+      python3
+      which
+    ]
     ++ lib.optional enableValgrindCheck valgrind
     ++ lib.optional fuseSupport fuse3
-    ++ lib.filter (lib.meta.availableOn stdenv.buildPlatform) [ erofs-utils fsverity-utils ];
+    ++ lib.filter (lib.meta.availableOn stdenv.buildPlatform) [
+      erofs-utils
+      fsverity-utils
+    ];
 
   preCheck = ''
     patchShebangs --build tests/*dir tests/*.sh
     substituteInPlace tests/*.sh \
       --replace " /tmp" " $TMPDIR" \
       --replace " /var/tmp" " $TMPDIR"
+    substituteInPlace tests/test-units.sh \
+      --replace 'BINDIR="$1"' 'BINDIR=$(realpath "$1")' \
+      --replace 'composefs-info' '$BINDIR/composefs-info'
```

Comment:
Far more important is to replace --replace with --replace-fail, to fail the build if nothing matches

Reply:
I actually contributed a PR to fix this upstream, I could just replace it with a patch pending the next version.